ts/packages/ts/src/stores/Configu.ts

```diff
@@ -36,6 +48,14 @@ export class ConfiguConfigStore extends ConfigStore {
   }
 
   async set(configs: Config[]): Promise<void> {
-    await this.client.put('/config', { configs });
+    const response = await this.client.put('/config', { configs });
+    if (response.status === 202) {
+      const protectedSet = response.data.diff.pending[0].set;
+      const queueUrl = `${response.data.queueUrl}?sp=${protectedSet}`;
+      throw new ConfiguConfigStoreApprovalQueueError(
```

Comment:
This error building should be made in the very specific error type you created `ConfiguConfigStoreApprovalQueueError`.
Personally I prefer just a "normal error here" as I can seem to think how should it be reused anywhere else.

Reply:
- Good point regarding the building of the error
- We need a unique error otherwise we will have to rely on the error message string to identify if this error was triggered by the approval queue and we don't want to do that.